engine/js/ng/sites/sections/site-sections.actions.js

```diff
@@ -60,55 +61,47 @@
     },
 
 
-    updateSection: function(path, value, onComplete) {
+    updateSiteSection: function(path, value, onComplete) {
       return function (dispatch, getStore) {
-        dispatch({ type: ActionTypes.UPDATE_SECTION });
+        dispatch({ type: ActionTypes.UPDATE_SITE_SECTION });
 
-        sync(window.Berta.urls.sections, { path: path, value: value })
+        sync(window.Berta.urls.site_sections, { path: path, value: value })
           .then(function (response) {
             if (response.error_message) {
               // @TODO dispatch error message
             } else {
-              dispatch(Actions.sectionUpdated(response));
+              dispatch(Actions.siteSectionUpdated(response));
             }
             onComplete(response);
           });
       };
     },
 
 
-    sectionUpdated: function(resp) {
+    siteSectionUpdated: function(resp) {
       return {
-        type: ActionTypes.SECTION_UPDATED,
+        type: ActionTypes.SITE_SECTION_UPDATED,
         resp: resp
       };
     },
 
 
-    renameSectionsSitename: function (data) {
-      return {
-        type: ActionTypes.RENAME_SECTIONS_SITENAME,
-        data: data
-      };
-    },
-
-
-    deleteSiteSections: function (data) {
+    renameSiteSectionsSitename: function (data) {
       return {
-        type: ActionTypes.DELETE_SITE_SECTIONS,
+        type: ActionTypes.RENAME_SITE_SECTIONS_SITENAME,
         data: data
       };
     },
 
 
-    resetSection: function(path, onComplete) {
+    resetSiteSection: function(path, onComplete) {
       return function (dispatch, getStore) {
         dispatch({
-          type: ActionTypes.RESET_SECTION,
+          type: ActionTypes.RESET_SITE_SECTION,
           path: path
         });
 
-        sync(window.Berta.urls.sections_reset, { path: path })
+        sync(window.Berta.urls.site_sections_reset, { path: path })
```

Comment:
Let's stick to camel case everywhere by default unless we have some specific reason not to use it (like constant names `THIS_IS_CONSTANT`)